lib/constants/styles.dart

```diff
@@ -143,6 +143,20 @@ class TextStyles {
         .labelLarge!
         .copyWith(fontWeight: FontWeight.w600);
   }
+
+  static TextStyle? bigButtonText(BuildContext context) {
+    ThemeData theme = Theme.of(context);
+    return theme.textTheme.titleMedium?.copyWith(
+      color: theme.colorScheme.primary,
+    );
+  }
+
+  static TextStyle? bigButtonDisabledText(BuildContext context) {
+    ThemeData theme = Theme.of(context);
+    return theme.textTheme.titleMedium?.copyWith(
+      color: theme.colorScheme.surfaceVariant,
```

Comment:
surfaceVariant is meant for surfaces not for text. In this particular case there is not enough contrast between the disabled button's color and the text color, so it is hard to read. You can try **onSurfaceVariant** for more readable text. 

In case you are curious about what these color names mean, the Material Design spec defines a set of color roles for surfaces and key elements, then there is a second set of colors that are meant to contrast with the first set when put on top (hence the "on" prefix). You can read more about color roles here if you are interested: https://m3.material.io/styles/color/the-color-system/color-roles

Reply:
Great read! Thanks for pointing this out. Changed to using onSurfaceVariant